crates/network-scanner/Cargo.toml

```
@@ -10,7 +10,7 @@ publish = false
 anyhow = "1"
 crossbeam = { version = "0.8", features = ["crossbeam-channel"] }
 dns-lookup = "2.0"
-mdns-sd = "0.10"
+mdns-sd = "0.11.0"
```

suggestion: Avoid specifying the patch number. We may over-constrain our dependency tree.
```suggestion
mdns-sd = "0.11"
```